### DISCUSSION TITLE
posts/win11-snap-groups-good/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#37] [Comments]
# posts/win11-snap-groups-good/

升级 Win11 后发现它的分屏和 Win10 有不同，由于一直用着双屏，也没感觉到明显改进。今年春节只带了笔记本回家，家里没有显示器，才发现 Snap groups 功能大大滴好。
[&hellip;] 简而言之，Snap groups 是把正在分屏的多个应用组合在一起变成任务栏里的一个窗口，激活这个窗口就可以回到原来的分屏组合。
[&hellip;] 笔记本一般会应用分屏，提高屏幕使用的效率。 &hellip;

https://shitao5.org/posts/win11-snap-groups-good/

**Comment by CyrusYip** [comment]
我也习惯用两个屏幕。最近买了便携显示器（799块钱/1440P/144Hz/15
.6寸），出门带着笔记本电脑和它，用得很爽。

**Reply by Shitao5** [reply]
哈哈之前我也看过便携显示器，但我出门少，觉得使用场景不多就没入手。
